HelloSP.cu

```diff
@@ -41,41 +41,6 @@ using namespace std;
 typedef unsigned int UInt;
 typedef float Real;
 
-UInt* generatePotentialPoolsWithVariableLength(UInt* potentialPools, int cols, const UInt IN_BLOCK_SIZE, Real potentialPct, const UInt MAX_CONNECTED)
-{
-    int connected = 0;
```

Comment:
thanks for the cleanup here in HelloSP! 
I did not understand well your comment "..are not duplicit, one is for initiation on device (kernel), one is for host.."? 
..so the removed code is not needed, right?

Reply:
Depends on what you mean by "needed" :) It is not used now, but can be used if the user wanted to try generating potential pools and permanences on the host in different ways. I will leave it on my branch to be able to show the differences.